### PR TITLE
Update colab wheel URL for colab tutorials.

### PR DESCRIPTION
**Pull request by yeounoh**
Use the colab 1.11 wheel for tutorials, since 2vm wheels are not compatible.

### CODE REVIEW

**Comment by JackCaoG** [comment]
Change is quite large, I thought this is just a wheel replacement?

File: contrib/colab/Distributing_Large_Embedding_tables.ipynb [hunk 1]
```diff
@@ -282,27 +270,27 @@
...
       ]
     },
     {
       "cell_type": "code",
+      "execution_count": null,
       "metadata": {
-        "id": "HCcPvjX36i_7",
+        "colab": {},
         "colab_type": "code",
-        "colab": {}
```
Comment (miaoshasha) [inline contrib/colab/Distributing_Large_Embedding_tables.ipynb:304]: just quick question, how did these changes happen?

Comment (yeounoh) [reply contrib/colab/Distributing_Large_Embedding_tables.ipynb:304]: > Change is quite large, I thought this is just a wheel replacement?

Yea, they are all URL path changes.

Comment (yeounoh) [reply contrib/colab/Distributing_Large_Embedding_tables.ipynb:304]: I only changed the pip url for torch_xla?

Comment (yeounoh) [reply contrib/colab/Distributing_Large_Embedding_tables.ipynb:304]: And yes, if you look just above, you would see the same elements are tehre, e.g., `"colab": {}`. They are reordered.

**Review by JackCaoG — APPROVED** [review]

**Pull request merged**